docs/MigrationGuides/migration_11_to_12.md

```diff
@@ -0,0 +1,17 @@
+# Migration Guide from 11.x to 12.x
+
+This project follows [Semantic Versioning](http://semver.org).
+
+### ReactiveSwift Migration
+- Check the [ReactiveSwift 4.0.0 release notes](https://github.com/ReactiveCocoa/ReactiveSwift/releases/tag/4.0.0) for changes related to ReactiveSwift.
+
+### Result Migration
+- Check the [Result 4.0.0 release notes](https://github.com/antitypical/Result/releases/tag/4.0.0) for changes related to Result.
+
+### AccessTokenPlugin Migration
+- Handle `.custom(String)` case for `AuthorizationType` or add a default case to achive exhaustiveness.
+- The `tokenClosure` parameter of the `AccessTokenPlugin` initializer is no longer an `@autoclosure` so you need to wrap the existing value in a closure.
+
+### Response Migration
```

Comment:
I would flip the order of this entire file TBH. 

Response, AccessTokenPlugin, and only then 3rd part extensions